lib/src/presentation/scenes/login/login.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+
+class LoginUi extends StatefulWidget {
```

Comment:
@Babiy7 let's use the same naming concept for classes.
Please rename it to "LoginScreen"